### PR TITLE
Fixes #24550: Nodes table should not be displayed in every tab of group details

### PR DESCRIPTION
**Pull request by clarktsiory**
https://issues.rudder.io/issues/24550
Peek 2024-03-27 16-55

### CODE REVIEW

**Comment by clarktsiory** [comment]
PR updated with a new commit

**Comment by clarktsiory** [comment]
PR updated with a new commit

**Comment by clarktsiory** [comment]
PR updated with a new commit

**Comment by clarktsiory** [comment]
PR updated with a new commit

**Comment by clarktsiory** [comment]
PR updated with a new commit

File: webapp/sources/rudder/rudder-web/src/main/scala/com/normation/rudder/web/components/SearchNodeComponent.scala [hunk 1]
```diff
@@ -356,7 +356,11 @@ class SearchNodeComponent(
     def showQueryAndGridContent(): NodeSeq = {
       (
         "content-query" #> NodeSeq.Empty
-        & "update-nodestable" #> srvGrid.displayAndInit(Some(Seq()), "groupNodesTable") // we need to set something, or IE moans
+        & "update-nodestable" #> srvGrid.displayAndInit(
+          Some(Seq()),
+          "groupNodesTable",
+          Some(showNodesTableByTab())
+        ) // we need to set something, or IE moans
```
Comment (fanf) [inline webapp/sources/rudder/rudder-web/src/main/scala/com/normation/rudder/web/components/SearchNodeComponent.scala:363]: I think that comment can be removed :)

**Review by VinceMacBuche — CHANGES_REQUESTED** [review]
The node list should only be displayed in Critera and summary at least that what i expected

**Comment by clarktsiory** [comment]
PR updated with a new commit

**Comment by clarktsiory** [comment]
OK, squash merging this PR

**Pull request merged**